SleepEngine/Demos/Bubbles/Entities/Bubble.cpp

```diff
@@ -33,8 +33,8 @@ void Bubble::update(float dt)
 std::unique_ptr<slp::Object> createBubbleObject(BubbleSettings settings)
 {
     auto bubble = slp::createGameObject();
-    bubble->addComponent(std::make_unique<Bubble>(settings));
-    auto* renderer = bubble->addComponent<slp::Renderer>();
+    bubble->addComponent<Bubble>(settings);
+    auto* const renderer = bubble->addComponent<slp::Renderer>();

```

Comment:
Taras, pls, let's use const for pointers. This is the same variables as any others (you use const for glm::vec2, float, but not for pointers??)